src/_P078_Eastron.ino

```diff
@@ -166,23 +157,63 @@ boolean Plugin_078(byte function, struct EventStruct *event, String& string)
     case PLUGIN_WEBFORM_LOAD:
       {
         {
-          String options_model[4] = { F("SDM120C"), F("SDM220T"), F("SDM230"), F("SDM630") };
-          addFormSelector(F("Model Type"), P078_MODEL_LABEL, 4, options_model, NULL, P078_MODEL );
+          String options_model[5] = { F("SDM220 & SDM120CT & SDM120"), F("SDM230"), F("SDM72D"), F("DDM18SD"), F("SDM630") };
```

Comment:
Changing order of model numbers will render exiting settings incompatible.
Better make a longer list of options to later let them use the same code.
If you want to change the order in the list, you can supply the optional array of option values along with the `addFormSelector`.